superbench/benchmarks/result.py

```diff
@@ -33,6 +33,7 @@ def __init__(self, name, type, return_code, run_count=0):
         self.__result = dict()
         self.__result['return_code'] = [return_code.value]
         self.__reduce_op = dict()
+        self.__reduce_op['return_code'] = None
```

Comment:
why it is None, not Last etc.?

Reply:
This is metric `return_code`, every worker should have its own return_code, don't need to do reduce.

Reply:
because it's a list, is it possible to have multiple return codes? e.g., benchmark itself add a metric with same name

Reply:
Here the list is for `run_count`. It is aligned with other metrics, but for `return_code`, we only have one for each BenchmarkResult instance. I think Benchmark itself shouldn't have metric named as `return_code`.